.zenodo.json

```diff
@@ -330,6 +330,11 @@
       "name": "Blair, Ross W.",
       "orcid": "0000-0003-3007-1056"
     },
+    {
```

Comment:
This gets autogenerated, you'll have to add yourself to this file https://github.com/nipreps/fmriprep/blob/master/.maint/CONTRIBUTORS.md

Reply:
Thanks, put the info in the right place this time.